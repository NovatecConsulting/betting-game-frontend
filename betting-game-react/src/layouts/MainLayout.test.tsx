import React from "react";
import { shallow } from "enzyme";
import { MainLayout } from ".";

describe(MainLayout.name + " component", () => {
    it("should render and match snapshot, given no children.", () => {
        const wrapper = shallow(<MainLayout />);

        expect(wrapper).toMatchSnapshot();
    });

    it("should render and match snapshot, given one child.", () => {
        const wrapper = shallow(
            <MainLayout>
                <div>Test</div>
            </MainLayout>
        );

        expect(wrapper.find("div")).toHaveLength(1);
        expect(wrapper).toMatchSnapshot();
    });
});
