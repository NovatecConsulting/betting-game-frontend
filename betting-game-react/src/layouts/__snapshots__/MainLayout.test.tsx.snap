// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainLayout component should render and match snapshot, given no children. 1`] = `
<Fragment>
  <Navbar />
</Fragment>
`;

exports[`MainLayout component should render and match snapshot, given one child. 1`] = `
<Fragment>
  <Navbar />
  <div>
    Test
  </div>
</Fragment>
`;
